documentation/modules/operators/proc-disabling-fips-mode-cluster-operator.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// assembly-using-the-cluster-operator.adoc
+
+[id='proc-disabling-fips-mode-cluster-operator-{context}']
+= Disabling FIPS mode
+
+[role="_abstract"]
+If you want, you can also disable the FIPS mode by setting the `FIPS_MODE` environment variable to `disabled` in the deployment configuration for the Cluster Operator.
```

Comment:
```suggestion
Strimzi automatically switches to FIPS mode when running on a FIPS-enabled Kubernetes cluster.
Disable FIPS mode by setting the `FIPS_MODE` environment variable to `disabled` in the deployment configuration for the Cluster Operator.
```

Reply:
Might be useful to say why you might want to do this?

Reply:
> Might be useful to say why you might want to do this?

To be honest, I do not know. But I want to keep the option _just in case_ and since the doc is already there, though tit might be easiest to keep it 🤷